easybuild/easyconfigs/s/SentencePiece/SentencePiece-0.2.0-GCC-12.3.0.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'Bundle'
+
+name = 'SentencePiece'
+version = '0.2.0'
+
+homepage = 'https://github.com/google/sentencepiece'
+description = "Unsupervised text tokenizer for Neural Network-based text generation."
+github_account = 'google'
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('gperftools', '2.12'),
```

Comment:
SentencePiece requires protobuf/24.0 as a dependency

Reply:
I don't understand how only you managed to trigger this. Also, i assume you meant protobuf-python/24.0

Reply:
No I mean
```
('protobuf', '24.0')
```

Reply:
protobuf-lite is included in sentencepiece as a thirdparty package (maybe we don't actually want that, but regardless), i would have expected it to find that, rather than having the python module link to something potentially different.

> It fails for me since I don't have stuff like that in my container :-)

And surely so does noone else on their OS either. I certainly don't have it, no trace of protobuf, protobuf-lite.


After some digging, I found that one can use external packages for Abseil and protobuf, using
```
'configopts': '-DSPM_PROTOBUF_PROVIDER=package -DSPM_ABSL_PROVIDER=package'
```
when attempting this, I also noticed that 
`'prebuildopts': 'export PKG_CONFIG_PATH=%(installdir)s/lib/pkgconfig:$PKG_CONFIG_PATH`
my pkgconfig stuff ended up in lib64, not lib. Fixing that made `sentencepiece` (the python part) not even look for protobuf. It shouldn't be looking for that stuff to start with, since it should just be using the already compiled `SentencePiece` from the first component. So, I think we have some lib/lib64 difference here. 

I attempted to use external protobuf and Abseil, but I got very very confusing problems. the ABSL_FLAG parsing completely broke, with strange broken output and warnings messages and strange output from even basic things like `spm_train --version`. I couldn't make any sense of it, so I reverted back. 

I've added pkgconf as a builddep and changed the PKG_CONFIG_PATH to include lib64 instead, which I hope will be correct for everyone? I'm not sure. Mixing in a different external protobuf is no good, because it seems to conflict with the bundled thirdparty abseil, and taking in both of them just broke things for me.